### PR TITLE
Si/feat/post section

### PR DESCRIPTION
**Pull request by siqbal181**

### CODE REVIEW

File: frontend/src/components/FeedColumn/FeedColumn.tsx [hunk 1]
```diff
@@ -1,17 +1,51 @@
 import "./FeedColumn.css";
-import { dummyFeedData } from "../../data/dummyFeedData";
-import FeedItem from "../FeedItem/FeedItem";
+import PostItem from "../Post/PostItem";
+import { useEffect, useState } from "react";
+import { PostProps } from "../../types/post.types";
+
+interface FeedColumnProps {
+  posts: PostProps;
```
Comment (siqbal181) [inline frontend/src/components/FeedColumn/FeedColumn.tsx:7]: Array of post props

File: frontend/src/components/FeedColumn/FeedColumn.tsx [hunk 2]
```diff
@@ -1,17 +1,51 @@
...
+}
+
+const FeedColumn: React.FC<FeedColumnProps & PostProps> = ({ posts }) => {
```
Comment (siqbal181) [inline frontend/src/components/FeedColumn/FeedColumn.tsx:10]: Don't pass it in as a prop

**Review by pablisch — COMMENTED** [review]
Just those things we discussed that you have made comments on. Let me know when they are done. Nothing has to be finished, I am keen to get this in main 👍🏼.

**Comment by siqbal181** [comment]
@pablisch All done! (and all ugly!)

**Review by pablisch — APPROVED** [review]
Looks good

**Pull request merged**